plugin/storage/es/spanstore/reader_test.go

```diff
@@ -668,11 +668,142 @@ func TestFindTraceIDs(t *testing.T) {
 	testGet(traceIDAggregation, t)
 }
 
-func TestFindTraceIDNotImplemented(t *testing.T) {
+func TestSpanReader_TestFindTraceIDs(t *testing.T) {
```

Comment:
These tests look very similar to the `TestSpanReader_TestFindTraces, ...` tests. Is there a way to DRY them up?